Data/Repositories/CustomerRepository.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Data.Entities;
+using Data.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Transactions;
+
+namespace Data.Repositories
+{
+    public class CustomerRepository : ICustomerRepository
+    {
+        private readonly List<Customer> _customers = new();
+
+        public void Create(Customer createdCustomer)
+        {
+            bool emailAlreadyExists = _customers.Any(customer => customer.Email == createdCustomer.Email);
+            if (emailAlreadyExists)
+            {
+                throw new ArgumentException("Email already exists");
+            }
+
+            bool cpfAlreadyExists = _customers.Any(customer => customer.Cpf == createdCustomer.Cpf);
+            if (cpfAlreadyExists)
+            {
+                throw new ArgumentException("Cpf already exists");
+            }
```

Comment:
```suggestion
            if (emailAlreadyExists)
                throw new ArgumentException("Email already exists");
```

Quando temos apenas uma instrução dentro de um IF podemos deixar sem os parenteses, mas devemos obedecer à indentação.